general/coding_standards.rst

```diff
@@ -42,6 +42,15 @@ Variables are initialized according to these general guidelines:
       which can be a scalar or a composite type are initialized with
       ``memset()`` in order to minimize the amount of future headache.
 
+Unsigned integer constants are defined using the ``U()`` macro or any of
+the minimum width cousins ``UINT{8,16,32,64}_C()``. This makes the sign and
```

Comment:
When should `UINT{8,16,32,64}_C` be used instead of `U{,L{,L}}()`?

Reply:
They are a bit more clumsy to use so I guess they are more useful to be compatible, on the other hand there's no harm in using them.